clang/tools/dpct/sycl-mapping-APIs-scanner.md

```diff
@@ -0,0 +1,26 @@
+# sycl-mapping-api-scanner
```

Comment:
If the dir only one md file, this md file maybe be shown in the github page. Can you help me to double check and confirm? @tomflinda 
I think it should be moved to the py file.

Reply:
@wenhuiNi confirmed for this "this md file maybe be shown in the github page as the Introduction of this folder"